CriteoPublisherSdk/Sources/CR_HeaderBidding.m

```diff
@@ -198,6 +202,14 @@ - (void)addCriteoBidToMopubRequest:(id)adRequest
     }
 
     if ([targeting isKindOfClass:[NSString class]]) {
+      NSString *displayUrl = bid.mopubCompatibleDisplayUrl;
+
+      // MoPub interstitial restrains itself to the safe area.
+      if (adUnit.adUnitType == CRAdUnitTypeInterstitial) {
```

Comment:
Unless mistaking this codepath isn't tested, is it?

Reply:
Yes it is there: https://github.com/criteo/ios-publisher-sdk/blob/586fd738b2e5a7144b8edde2bef62b8d0155648b/CriteoPublisherSdk/Tests/UnitTests/CR_HeaderBiddingTests.m#L235

Reply:
I meant the whole branch, are you checking that interstitials get their displayUrl altered?

Reply:
Yes it is, the injector is stubbed to return a display.url value when getting the bid display URL. Then this display.url value  is checked.